web/models/entry.ex

```diff
@@ -192,8 +194,13 @@ defmodule SignDict.Entry do
     end
   end
 
-  def url(entry) do
-    "url"
+  def set_url(nil), do: nil
+  def set_url(entry = %Entry{}) do
+    entry |> Map.merge(%{url: entry |> url()})
+  end
+
+  def url(entry = %Entry{}) do
+    "#{Application.get_env(:sign_dict, SignDict.Endpoint)[:url][:host]}/entry/#{entry.id}"
```

Comment:
I think this way the protocol is missing in the URL? Should we prepend `https://` to make it work?

Reply:
ah yes of course.